_config.yml

```diff
@@ -18,7 +18,7 @@ email: msse@berkeley.edu
 description: >- # this means to ignore newlines until "baseurl:"
   A website for the MSSE Bootcamp
 repository: 
-baseurl: "" # the subpath of your site, e.g. /blog
+baseurl: "blog-scott-s-to/" # the subpath of your site, e.g. /blog
```

Comment:
I am guessing we only need to post one of these. It looks like the name is wrong here so I can submit a second file if you want to review it and merge the changes.

Reply:
Sure we can do that

Reply:
Should probably revert this pull request.